### PR TITLE
Optimization and better syntax

### PR DESCRIPTION
**Pull request by oshaulz**
Added auto config, customizable status and stable default arguments

### CODE REVIEW

File: config.json.example [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
     "token": " --TOKEN HERE-- ",
-    "prefix": "."
+    "prefix": ".",
+    "status": "The game of life"
```
Comment (hydrostaticcog) [inline config.json.example]: Change this to something more McPy-y.

**Review by hydrostaticcog — COMMENTED** [review]
Looks fine. Just please change the default status to be better.

**Comment by oshaulz** [comment]
Done! It should display "Playing **on the McPy server"**

**Review by hydrostaticcog — APPROVED** [review]
Looks good! Will be pushed onto the production bot soon.

**Pull request merged**